pkg/protocol/amqp/amqp_test.go

```diff
@@ -44,7 +45,7 @@ var (
 	ceTimestamp     = cetypes.Timestamp{Time: time.Date(2020, 03, 21, 12, 34, 56, 780000000, time.UTC)}
 	cneTimestamp    = types.Timestamp{Time: time.Date(2020, 03, 21, 12, 34, 56, 780000000, time.UTC)}
 	ceSchema        = cetypes.URI{URL: url.URL{Scheme: "http", Host: "example.com", Path: "/schema"}}
-	_type           = "ptp_status_type"
+	_type           = ptp.STATUS
```

Comment:
add real type after refactoring and remove ptp.STATUS and ptp.EVENT

Reply:
done